{"name": "test_sub[int]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 9 == 10\n  +9\n  -10", "trace": "self = <test_calc.TestCalc object at 0x106172910>, a = 10, b = 1, expect = 9, myfixture = None\n\n    @pytest.mark.run(order=3)   # 用例排序\n    @pytest.mark.flaky(retuns=3, reruns_delay=2)    # 失败后重新运行\n    @pytest.mark.parametrize('a,b,expect', get_datas()[1], ids=get_datas()[4])  # 数据驱动\n    def test_sub(self, a, b, expect, myfixture):\n>       assert expect == a - b + 1\nE       assert 9 == 10\nE         +9\nE         -10\n\ntest_calc.py:35: AssertionError"}, "parameters": [{"name": "a", "value": "10"}, {"name": "b", "value": "1"}, {"name": "expect", "value": "9"}], "start": 1608120998263, "stop": 1608120998263, "uuid": "9f03d72e-262d-44d3-b4a1-196741c9ec16", "historyId": "3a087d2fbb54ccefdcf229004bceee2f", "testCaseId": "3b960671b7ce69bb509e37e84b3024bc", "fullName": "test_calc.TestCalc#test_sub", "labels": [{"name": "tag", "value": "run(order=3)"}, {"name": "tag", "value": "flaky(retuns=3, reruns_delay=2)"}, {"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "bogon"}, {"name": "thread", "value": "92175-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}